config.ru

```diff
@@ -1,5 +1,10 @@
 require 'mumukit'
 
+module Mumukit::WithCommandLine
+  def limit_command
+    '.heroku/vendor/bin/limit'
```

Comment:
Can't we include this in Mumukit? Having to include it in every runner is a bit against DRY.